plugins/output/animationAssetUnpack/index.js

```diff
@@ -0,0 +1,277 @@
+                if(!err) {
+                  // removes redundant root folder from
+                  // the OAM package file
+                  removePackageRoot(packageDir, filesFromPackage);
+
+                  // removes the oam package, not
+                  // needed in exported course
+                  fsextra.removeSync(packageFileName);
+
+                  unpackedCount++;
+
+                  if (unpackedCount === total) {
+                    onUnpackDone(null, filesFromPackage);
+                  }
+                }
+                else {
+                  logger.log('error', ERROR_UNPACKING, err);
+                  onUnpackDone(err, null);
+                }
+              });
+
+          })
+          .catch(function(err) {
+            logger.log('error', ERROR_UNPACKING, err);
+            onUnpackDone(err, null);
+          });
+      })();
+    }
+  },
+
+  /**
+   * Watch the course build folder where
+   * Adapt AT works to preview the course.
+   * Checks presence of OAM/ZIP files and unpacks
+   * them if any.
+   * @param watchingDir {string}
+   */
+  watchCourseOutputDir: function(watchingDir) {
+    var monitor = fsmonitor.watch(watchingDir, {
+      matches: function(relpath) {
+        return (
+          (relpath.match(/\.oam$/i) !== null)
+        );
+      },
+      // exclude not needed directories
+      // TODO: periodically review this list on new AT releases
+      excludes: function(relpath) {
+        return (
+          relpath.match(/^\.git$/i) !== null ||
+          relpath.match(/^\node_modules$/i) !== null ||
+          relpath.match(/^\grunt/i) !== null ||
+          relpath.match(/^\src/i) !== null
+        );
+      }
+    });
+
+    monitor.on('change', function onChange(changes) {
+      var packagesList = [];
+
+      for (var i in changes.addedFiles) {
+        packagesList.push(path.join(watchingDir, changes.addedFiles[i]));
+      }
+      for (var ii in changes.modifiedFiles) {
+        packagesList.push(path.join(watchingDir, changes.modifiedFiles[ii]));
+      }
+      //
+      // Monitor.on('change') also returns "changes.removedFiles"
+      // but we don't care about them since here we
+      // are working with the course preview only,
+      // and the output doesn't go into the course
+      // export downloaded by the user.
+      pluginInstance.unpack(packagesList, function onUnpacked(err) {
+        if (err) {
+          logger.log('error', ERROR_UNPACKING, err);
+        }
+      });
+    });
+  }
+};
+
+
+/**
+ * Remove the redundant package
+ * root generated by decompressing
+ * the OAM/ZIP file. All included files
+ * are moved one level up in the
+ * directory tree.
+ * @param packageDir {string}
+ * @param filesFromPackage {object[]}
+ */
+function removePackageRoot(packageDir, filesFromPackage) {
+  var entries = fs.readdirSync(packageDir);
+  // there is a root in decompressed package...
+  if (entries.length === 1) {
+    var packageRoot = entries[0] + '/';
```

Comment:
Hi @fabiobeoni , the separator does not cover Windows-based AT, [path.sep](https://nodejs.org/api/path.html#path_path_sep) would be needed here.
Recently I'm trying out your PR with [adapt-animation-frame](https://github.com/fabiobeoni/adapt-animation-frame), it works smoothly together!

Reply:
Thanks for your note, sometime I would remove the "/" button on my keyboard ;)  Let's change it.